src/sshping.cxx

```diff
@@ -127,13 +142,100 @@ void die(const char* msg) {
     exit(255);
 }
 
+#if _WIN32
+  #include <windows.h>
+  #include <conio.h>
+  #ifndef PASS_MAX
+    #define PASS_MAX 512
+  #endif
+
+double PCFreq = 0;
+
+// Replacement for the getpass UNIX method
+char *getpass(const char *prompt) {
+	char getpassbuf[PASS_MAX + 1];
```

Comment:
In fact if you put:

`const int pass_max = 512;`

in here at the top altogether and removed PASS_MAX it would make more sense. No need for global scope.

For global constant values in C++ in general `static const whatevertype = whatevervalue;` is preferred. Here we don't need a global variable at all!